en/telemetry/crossfire_telemetry.md

```diff
@@ -0,0 +1,165 @@
+
+### Parameter Configuration
+
+[Set the parameter](../advanced_config/parameters.md) named [RC_CRSF_PRT_CFG](../advanced_config/parameter_reference.md#RC_CRSF_PRT_CFG) to port that is connected to the CRSF receiver (such as `TELEM1`).
+
+This [configures the serial port](../peripherals/serial_configuration.md) to use the CRSF protocol.
+Note that some serial ports may already have a [default serial port mapping](../peripherals/serial_configuration.md#default-serial-port-configuration) or [default MAVLink serial port mapping](../peripherals/mavlink_peripherals.md#default-mavlink-ports) that you will have to un-map before you can assign to CRSF.
+For example, if you want to use `TELEM1` or `TELEM2` you first need to modify [MAV_0_CONFIG](../advanced_config/parameter_reference.md#MAV_0_CONFIG) or [MAV_1_CONFIG](../advanced_config/parameter_reference.md#MAV_1_CONFIG) to stop setting those ports.
+
+There is no need to set the baud rate for the port, as this is configured by the driver.
+
+## Radio Setup
+
+See [TBS Crossfire Manual](https://www.team-blacksheep.com/tbs-crossfire-manual.pdf) (search for 'Setting up radio for CRSF').
+
+<a id="transmitters"></a>
+
+## Compatible RC Transmitters
+
+:::note
+An RC radio system historically consisted of a handheld controller on the ground that transmitted to an on-vehicle receiver.
+Even though many radio systems are now bidirectional, the ground module is often referred to as the transmitter, and the air unit is called a receiver.
+:::
+
+You will need an RC transmitter that can receive the CRSF telemetry stream from a bound CRSF receiver on the vehicle.
+
+Radio controllers may come with an inbuilt base module ("transmitter") [such as these by TBS](https://www.team-blacksheep.com/shop/q:radio%20drone%20controller)
+
+Other ground controllers allow transmitters to be added as separate transmitter modules.
+TBS transmitters are usually compatible with the "JR module bay".
+For example the [FrSky Taranis X9D Plus (2)](https://www.frsky-rc.com/product/taranis-x9d-plus-2/) can be used with TBS modules
+
+Popular alternatives that can be used with [TBS Transmitter Modules](http://team-blacksheep.com/shop/cat:rc_transmitters#product_listing) include:
+
+- FrSky Taranis X9D Plus (recommended)
+- ?
+
+These transmitters will need to be configured for use with your transmitter.
+The [TBS Crossfire Manual](https://www.team-blacksheep.com/tbs-crossfire-manual.pdf) provides some common setup instructions (i.e. for OpenTX running on the Taranis X9D plus).
+
+<a id="receivers"></a>
+
+## Telemetry Receivers
+
+<!-- what should we list here? -->
+
+- [TBS Crossfire Nano RX](http://team-blacksheep.com/products/prod:crossfire_nano_rx) - designed for small Quads.
+- ELRS RX <!-- add link and more detail -->
+
+## Wiring
```

Comment:
@chris1seto

1.  Is there any difference between using a ELRS or TBS receivers/transmitters?
2. Are the ELRS/TBS receiver/transmitters compatible? I.e. can they be mixed?
3. I read that TBS use uninverted signals on UART which means that they can work on the older Pixhawks - which support uninverted signals - without needing inverter circuitry. Simpler cables!
   - Is this true?
   - Is it also true for ELRS? I
   - Note, I assume this is just a hardware 'benefit" of using TBS - so you could have CRSF inverted if you wanted.
4. Did CRSF used to be plug and play/part of rcin before you fixed it? I ask because [this doc says it is plug and play](https://docs.px4.io/main/en/flight_controller/omnibus_f4_sd.html#setup)  
5. I say below that you connect RX/TX on your flight controller to separate channels, because that is what was done in [this setup](https://docs.px4.io/main/en/flight_controller/omnibus_f4_sd.html#setup) .
   - How does this work? Do you have to setup the receiver to tell it that Ch1/Ch2 correspond to particular UART pins?